python/ray/ml/train/integrations/huggingface/huggingface_torch_trainer.py

```diff
@@ -0,0 +1,217 @@
+from typing import Any, Callable, Optional, Dict, Union, Type
+
+from transformers.trainer import Trainer
+
+from ray.train.torch import TorchConfig
+from ray.ml.trainer import GenDataset
+from ray.ml.train.integrations.torch import TorchTrainer
+from ray.ml.config import ScalingConfig, RunConfig
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.checkpoint import Checkpoint
+from ray.util import PublicAPI
+
+
+def _huggingface_train_loop_per_worker(config: Dict[str, Any]):
+    config
+    return
+
+
+@PublicAPI(stability="alpha")
+class HuggingFaceTorchTrainer(TorchTrainer):
```

Comment:
I would say let's just name this `HuggingFaceTrainer` or `HFTrainer`, just for consistency with Huggingface. Even for Huggingface the regular `Trainer` refers to Pytorch, and `TFTrainer` refers to Tensorflow.

Reply:
It might be worth taking a look at the conventions used by SageMaker for [Hugging Face models and estimators](https://sagemaker.readthedocs.io/en/stable/frameworks/huggingface/sagemaker.huggingface.html#).

(`sagemaker.huggingface.HuggingFace` and `sagemaker.huggingface.model.HuggingFaceModel` for training, and a simpler abstraction -- `sagemaker.transformer.Transformer` -- for inference)